<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fizClientView</class>
 <widget class="QDialog" name="fizClientView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Личные данные</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Фамилия</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineSurname"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Имя</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineName">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Отчество</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineMiddle"/>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Дата рождения</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="customDateEdit" name="dateBirthday"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QPushButton" name="buttonCancelFiz">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Отмена</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4" rowspan="2" colspan="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Контактные данные</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Телефон раб.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineTelW">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Телефон моб.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineTelM"/>
      </item>
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Адрес эл. почты</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEmail"/>
      </item>
      <item>
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Индекс</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineIndex"/>
      </item>
      <item>
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>Город</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineCity"/>
      </item>
      <item>
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>Улица</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineStreet"/>
      </item>
      <item>
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>Дом</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineBld"/>
      </item>
      <item>
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>Корпус</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineCorp"/>
      </item>
      <item>
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>Квартира</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineFlat"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QPushButton" name="buttonSaveFiz">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Сохранить</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Паспортные данные</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Серия</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="linePasSer"/>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Номер</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="linePasNum"/>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Кем выдан</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineIsByWhom"/>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Когда выдан</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="customDateEdit" name="dateWhenIs"/>
      </item>
      <item>
       <widget class="QLabel" name="label_100">
        <property name="text">
         <string>Код подразделения</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="linePasCode"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" rowspan="2" colspan="4">
    <widget class="QTextEdit" name="textNotes"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>customDateEdit</class>
   <extends>QDateEdit</extends>
   <header>customdateedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
